storage/perfschema/unittest/pfs_server_stubs.cc

```diff
@@ -44,3 +44,9 @@ void sql_print_warning(const char *format, ...)
 {
   /* Do not pollute the unit test output with annoying messages. */
 }
+
+my_bool
+acl_is_utility_user(const char *user, const char *host, const char *ip)
```

Comment:
Please, double check this does not trigger any `unused arguments` warnings. This concerns 5.6 as well